inferlo/pairwise/optimization/convex_bounds_test.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+
+from inferlo import PairWiseFiniteModel
+from inferlo.pairwise.optimization.convex_bounds import LPrelaxation
+from inferlo.pairwise.testing import grid_potts_model, tree_potts_model, \
+    line_potts_model
+
+
+def test_grid_4x4x2():
+    model = grid_potts_model(4, 4, al_size=2, seed=0)
+    max_lh_gt = model.max_likelihood(algorithm='bruteforce')
+    max_lh_gt_value = np.log(model.evaluate(max_lh_gt))
+    max_lh_value_relax = LPrelaxation(model)
+    assert (max_lh_gt_value <= max_lh_value_relax)
+
+
+def test_grid_3x3x4():
+    model = grid_potts_model(3, 3, al_size=4, seed=0)
+    max_lh_gt = model.max_likelihood(algorithm='bruteforce')
```

Comment:
How long does it take (PyCharm shows that)? If it's ~10ms it's fine. If it's seconds, please use more efficient algorithm for computing ground truth - path_dp or junction_tree.

We will have hundreds of tests and we will be running all of them on every merge, so we want to keep them no longer than needed to test correctness.